docs/examples/compute/cloudscale/cloudscale_compute_simple.py

```diff
@@ -0,0 +1,26 @@
+from pprint import pprint
+
+import libcloud
+
+cls = libcloud.get_driver(
+    libcloud.DriverType.COMPUTE,
+    libcloud.DriverType.COMPUTE.CLOUDSCALE
+)
+
+driver = cls('3pjzjh3h3rfynqa4iemvtvc33pyfzss2')
```

Comment:
I'd store the token string in a variable `token` to clarify what that string is.